tools/tsp-client/test/commands.spec.ts

```diff
@@ -400,33 +405,113 @@ describe.sequential("Verify commands", () => {
         ],
         emitterPackageJsonPath: "tools/tsp-client/test/utils/emitter-package.json",
       };
-      await mkdir(
-        joinPaths(
+      await mkdir(libraryPath, { recursive: true });
+      await writeTspLocationYaml(existingTspLocation, libraryPath);
+      // Now run the init command with --update-if-exists with a local spec so that we can pass in a dummy commit for testing
+      const outputDir = await initCommand(args);
+      const tspLocation = await readTspLocation(outputDir);
+      assert.deepEqual(tspLocation, {
+        directory: "specification/contosowidgetmanager/Contoso.WidgetManager",
+        commit: "abc",
+        repo: "Azure/azure-rest-api-specs",
+        additionalDirectories: [
+          "tools/tsp-client/test/examples/specification/contosowidgetmanager/Contoso.WidgetManager.Shared",
+        ],
+        emitterPackageJsonPath: "tools/tsp-client/test/utils/emitter-package.json",
```

Comment:
Is this the default emitterPackageJson? If it is, can we add another test case where there is a different existing emitterPackageJsonPath to verify it isn't changed?

Reply:
Yep, makes sense, thanks!